vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -211,6 +211,65 @@ public FormLayout() {
     public FormLayout(Component... components) {
         add(components);
     }
+    
+    /**
+     * Sets the colspan of the given component's element.
+     * You can directly add components with the wanted colspan with
+     * {@link #add(Component)}.
+     *
+     * @param component
+     *            the component to modify
+     *
+     * @param colspan
+     *            the desired colspan for the component
+     *
+     */
+    public void setColspan(Component component, int colspan) {
+    	String strColspan = "";
+    	if (colspan < 1) {
+    		strColspan = "1";
+    	} else {
+    		strColspan=String.valueOf(colspan);
+    	}
+    	component.getElement().setAttribute("colspan", strColspan );
+    }
+    
+    /**
+     * Adds a component with the desired colspan.
+     *
```

Comment:
Could add here a mention that "This method is a shorthand for calling #add(Component) and #setColspan(Component, int)"